manager/integration/tests/common.py

```diff
@@ -3517,26 +3532,32 @@ def get_minio_api(client, core_api, minio_secret_name):
                  secure=True)
 
 
-def minio_delete_random_backup_block(client, core_api, volume_name):
-    backup_target_credential_setting = client.by_id_setting(
-            SETTING_BACKUP_TARGET_CREDENTIAL_SECRET)
-
-    secret_name = backup_target_credential_setting.value
-
-    minio_api = get_minio_api(client, core_api, secret_name)
-
+def minio_get_volume_backup_prefix(volume_name):
     volume_name_sha512 = \
         hashlib.sha512(volume_name.encode('utf-8')).hexdigest()
 
-    bucket_name = "backupbucket"
     backup_prefix = "backupstore/backupstore/volumes"
```

Comment:
I think you need to pass the client and use get_backupstore_bucket_name ?

Reply:
Or the s3 url? with the /path?

Reply:
I've already added that: [here](https://github.com/meldafrawi/longhorn-tests/blob/bd3ff1c8bfa851e5395b30a1e41ff02ad93188b0/manager/integration/tests/common.py#L3559)

Reply:
I meant in regard to the backup_prefix = backupstore/backupstore/volumes
isn't the first /backupstore part of the s3 url?

Reply:
ie for the engine I switched the target to /s3test

Reply:
fixed